lib/matplotlib/backends/backend_ps.py

```diff
@@ -1442,7 +1445,7 @@ def convert_psfrags(tmpfile, psfrags, font_preamble, custom_preamble,
     latexfile = latexfile.replace("~", "\\string~")
     command = [str("latex"), "-interaction=nonstopmode",
                '"%s"' % latexfile]
-    verbose.report(command, 'debug')
+    log.debug('%s', command)
```

Comment:
we don't have to do it in this PR but I think we should just have our own wrapper around subprocess which does the logging every time we shell out